src/app/pages/StoryPage/index.test.jsx

```diff
@@ -40,6 +40,11 @@ jest.mock('#containers/ComscoreAnalytics', () => {
   return ComscoreAnalytics;
 });
 
+jest.mock('#containers/Ad', () => {
+  const AdsContainer = () => <div>STY ADS</div>;
+  return AdsContainer;
+});
+
```

Comment:
Can we add some coverage for application of the `allowAdvertising` flag to the Story page (https://github.com/bbc/simorgh/pull/7582/files#diff-1bb3c5d9c5f696f7cf7dbef26db8220eR146-R151) It should probably check the leaderboard and MPU ads containers are rendered when the flag is true.

Reply:
A probably better alternative is to add this coverage into integration tests as has been done for the FrontPage: https://github.com/bbc/simorgh/blob/latest/src/integration/pages/frontPage/canonicalTests.js#L28..L43